_commands/tools/units.md

```diff
@@ -3,22 +3,37 @@
 
 units
 ---
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+`units` converts quantities from one unit to another unit. It can be used interactively or non-interactively.
+
 
-<!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ units '10 meters' 'feet'
+# displays * 32.808399 
+           / .03048
 ~~~
 
 <!--more-->
 
 ### Useful Options / Examples
```

Comment:
there are a couple flags you can specify, namely `--digits` and `--verbose`. Try to include an example of them